### PR TITLE
AAP-17672 upated numbering, and added content.

### PR DESCRIPTION
**Pull request by rogrange**
Added a parapragh to section 4.12 "Building an ee in a disconnected environment." Fixed a numbering issue in section 4.12.1.

Resolves: AAP-17672
See also: #1058, #1059

### CODE REVIEW

File: downstream/modules/platform/con-building-an-execution-environment-in-a-disconnected-environment.adoc [hunk 1]
```diff
@@ -16,33 +16,11 @@ link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_pl
...
-//* `registry.connect.redhat.com`
-* `sso.redhat.com`
-* `cdn-ubi.redhat.com`
-* `cdn.quay.io`
-* `cdn01.quay.io`
-* `cdn02.quay.io`
-* `cdn03.quay.io`
-
-For additional information, see link:https://access.redhat.com/articles/1525183[Public CIDR Lists for Red Hat].
+Building an execution environment image in a disconnected environment requires mirroring content from these locations. See link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html-single/red_hat_ansible_automation_platform_installation_guide/index#importing-collections-into-private-automation-hub_disconnected-installation[Importing Collections into private automation hub] for information on importing collections from Ansible Galaxy or automation hub into a private automation hub.
```
Comment (hherbly) [inline downstream/modules/platform/con-building-an-execution-environment-in-a-disconnected-environment.adoc:19]: The first line is missing an attribute. "Building an {ExecEnvShort} image in a disconnected environment..."

Also, a couple of missed attributes after the link: ..."for information on importing collections from {Galaxy} or {HubName} into a {PrivateHubName}."

File: downstream/modules/platform/con-building-an-execution-environment-in-a-disconnected-environment.adoc [hunk 2]
```diff
@@ -16,33 +16,11 @@ link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_pl
...
 
 Mirrored PyPI content once transferred into the disconnected network can be made available using a web server or an artifact repository like Nexus.  The RHEL and UBI repository content can be exported from an internet-facing Red Hat Satellite server, copied into the disconnected environment, then imported into a disconnected Satellite so it is available for building custom {ExecEnvShort}s.  See link:https://access.redhat.com/documentation/en-us/red_hat_satellite/6.14/html-single/installing_satellite_server_in_a_disconnected_network_environment/index#iss_export_sync_in_an_air_gapped_scenario[ISS Export Sync in an Air-Gapped Scenario] for details.
```
Comment (hherbly) [inline downstream/modules/platform/con-building-an-execution-environment-in-a-disconnected-environment.adoc:21]: I think I see extra spaces at the end of each sentence in this paragraph--if not please disregard.

**Review by hherbly — APPROVED** [review]
I spotted a couple of missed attributes and extra spaces, but other than that LGTM! Approved.

File: downstream/modules/platform/con-building-an-execution-environment-in-a-disconnected-environment.adoc (hunk 1 above)
Comment (rogrange) [reply downstream/modules/platform/con-building-an-execution-environment-in-a-disconnected-environment.adoc]: I have no idea how that happened. They were in the original copy. Nice catch! Thanks!

**Pull request merged**